core/src/main/java/jenkins/agents/CloudSet.java

```diff
@@ -226,7 +226,6 @@ private void handleNewCloudPage(Descriptor<Cloud> descriptor, String name, Stapl
         checkName(name);
         JSONObject formData = req.getSubmittedForm();
         formData.put("name", name);
-        formData.put("cloudName", name); // ec2 uses that field name
```

Comment:
Removed use of `cloudName` from `AmazonEC2Cloud` in ec2: https://github.com/jenkinsci/ec2-plugin/pull/888

Reply:
Turns out this is probably not a good idea as it requires too many downstream changes, not only in ec2, put also casc. Other plugins like `azure-vm-agents` also use `cloudName` in the constructor signature.